cep-XXXX.md

```diff
@@ -0,0 +1,265 @@
+- `doc_url: str`. URL pointing to the documentation website of the package.
+- `env_vars: dict[str, str]`. Allow-listed environment variables set during the package build.
+- `extra: dict[str, Any]`: A free-form dictionary of arbitrary metadata. This MAY be used to record provenance metadata as described in [CEP PR#113](https://github.com/conda/ceps/pull/113).
+- `home: str`. URL pointing to the homepage of the package.
+- `license: str`. SPDX license identifier for the package.
+- `summary: str`: A short summary of the package (usually one sentence).
+
+These keys are commonly found, but are now considered deprecated:
+
+- `license_file: str | list[str]`: Paths to the license files, relative to the `recipe/` directory.
+- `license_family: str`. Use `license` instead.
+- `tags: list[str]`.
+- `identifiers: list[str]`.
+- `keywords: list[str]`.
+
+Additional keys MAY be allowed. Some examples include vendor information such as the conda-build and conda versions used at build time, along with the packages present in the conda `base` environment (`conda_build_version`, `conda_version`, `root_pkgs`, respectively).
+
+#### `./info/link.json`
+
+Optional.
+
+A dictionary containing important metadata for installation (linking) operations. Its contents MUST conform to the following schema:
+
+- `package_metadata_version: int`. Version of this file. The only known value is `1`.
+- `noarch: dict[str, Any]`. A dictionary describing the `noarch` configuration of the package. Its keys MUST conform to:
+  - `type: Literal["generic", "python"]`. Type of `noarch` package.
+  - `entry_points: list[str]`. If `type == "python"`, this key MUST list the console entry points of the Python package, as a list of strings with syntax `{executable-name}={dotted-import-path}:{function-name}` (for example `bsdiff4 = bsdiff4.cli:main_bsdiff4`).
+- `preferred_env: dict[str, str]`. Deprecated, optional. It MUST have two keys: `name: str` (name of the preferred environment) and `executable_paths: list[str]` (list of preferred executable paths in that environment).
+
+#### `./info/licenses/`
+
+Optional.
+
+Directory containing all the license files that apply to the distributed code and resources.
+
+It MAY be empty or absent if the recipe doesn't ship any installable files.
+
+#### `./info/recipe/`
+
+Optional.
+
+Directory containing the recipe used to build the package. It SHOULD contain the original and rendered versions of the recipe file, along with the files necessary to run the build again.
+
+It SHOULD contain a file specifying the licensing details of the recipe itself.
+
+#### `./info/test/`
+
+Optional.
+
+Directory containing the scripts and files needed to test the package.
+
+#### `./info/git`
+
+Optional.
+
+If the package sources come from a `git` repository and the `git` executable is available, this file will include information about the status of the cloned repository. Namely, the output of:
```

Comment:
Is there a stict requirement for the format of this file? Is it valid if the file exists but is empty?